pkg/plugins/utils/docker/dockerregistry/main.go

```diff
@@ -67,8 +67,13 @@ func (dgr DockerGenericRegistry) Digest(image dockerimage.Image) (string, error)
 		return "", err
 	}
 
+	// Provide the registry with a list of content types that we support:
+	//  * Docker Registry V2 manifest list
+	//  * List of OCI manifests (multiple architectures provided by the registry)
+	//  * Standalone OCI manifest (only one architecture provided by the registry)
 	req.Header.Add("Accept", "application/vnd.docker.distribution.manifest.list.v2+json")
-	req.Header.Add("Accept", "application/vnd.docker.distribution.manifest.v2+json")
```

Comment:
It seems that if we remove this content-type then dockerhub return v1 manifest instead.